pocld/virtual_cl_context.cc

```diff
@@ -779,7 +779,7 @@ void VirtualCLContext::BuildOrLinkProgram(Request *req, Reply *rep,
 
       std::vector<unsigned char> binary(buf, buf + bin_size);
       buf += bin_size;
-      assert((buf - buffer) <= (size_t)m.payload_size);
+      assert((size_t)(buf - buffer) <= (size_t)m.payload_size);
```

Comment:
FWIW, maybe use `static_cast` here instead of C-style cast.

Reply:
The whole file uses C-style casts, I did not want to go against the current pattern.

Reply:
Fair enough.